mu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+if [ $(id -u) != "0" ]
+then
+	echo "Please run this script as root"
+	echo "For installation instructions, please refer to readme.txt"
+	exit 1
+fi
+
+rm -rf /usr/local/share/wayland-sessions/miriway.desktop
+rm -f /usr/local/bin/miriway*
+rm -r /usr/local/etc/xdg/xdg-miriway
```

Comment:
On general principles I don't like duplicating the list of files, and this also assumes that `CMAKE_INSTALL_PREFIX` was set to `/usr/local/` (which is the default, but need not be the case).

What I tend to do, and works for all the cmake projects i deal with, is:

```bash
cat cmake-build-release/install_manifest.txt | xargs sudo rm
```

That does need a knowledge of the build directory, so using this in a script would require care.

Reply:
Implementing this will take some time, in the meantime any tips are welcome...

Reply:
> Implementing this will take some time, in the meantime any tips are welcome...

I don't think this is a problem worth solving. This script is only useful when the user is:

1. building and installing locally;
2. not familiar with cmake's `install_manifest.txt`; and,
3. more trusting of an `uninstall-miriway.bash` than in doing things manually.

Adding complexity to the script will only reduce the level of trust.

It is probably better to document how to uninstall miriway using the `install_manifest.txt` in README.md